_posts/2018-05-08-CocoaPods-1.6.0-beta.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "CocoaPods 1.6.0 Beta Begins!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.6.0_ release will focus on stability, performance and scalability for large projects through a longer beta cycle.
```

Comment:
The phrase at the end makes the first sentence awkward. I suggest moving `through a longer beta cycle` to another sentence.